controllers/agentcluster_controller.go

```diff
@@ -46,6 +50,7 @@ type AgentClusterReconciler struct {
 //+kubebuilder:rbac:groups=capi-provider.agent-install.openshift.io,resources=agentclusters/finalizers,verbs=update
 //+kubebuilder:rbac:groups=hive.openshift.io,resources=clusterdeployments,verbs=get;list;watch;create;update;patch;delete
 //+kubebuilder:rbac:groups=extensions.hive.openshift.io,resources=agentclusterinstalls,verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups=hypershift.openshift.io,resources=hostedcontrolplanes,verbs=get;list;watch;
```

Comment:
Will this fail if hypershift isn't installed?

Reply:
i don't know to say the truth but without it you can't reach the resource

Reply:
added junk resource and it looks that it passes